### ISSUE TITLE
Console shows Failed to spawn hologram but hologram is spawned

### ISSUE THREAD

**Issue by Iwitrag** [sainttx/Holograms#75]
Holograms version 2.9.1-SNAPSHOT
Paper version: git-Paper-171 (MC: 1.14.4) 

I'm using custom plugin to load custom world called 'pvp'. This world contains Holograms.
I've ensured to load my plugin BEFORE Holograms so Holograms knows about this world before actually loading holograms.

I have this in my plugin.yml

```
loadbefore:
  - Holograms
```

this is my startup log: https://pastebin.com/CTFmaYah

As you can see - my plugin GreenCore [CORE] starts before Holograms and loads world 'pvp' before Holograms actually tries to put holograms into 'pvp'.

But still - Holograms shows this:

```
[10:05:08 WARN]: [Holograms] Failed to spawn hologram entity in world pvp at x:-209.47680412625456 y:98.20000004768372 z:-145.43121134628277
[10:05:08 INFO]: [Holograms] Loaded hologram with "parkour_event_01_winner" with 1 lines
```

Hologram is correctly spawned but the warning is annoying.

**Comment by Iwitrag** [comment]
Also when this happens - all hologram lines are placed "inside" each other... I need to reload plugin to fix that

**Comment by Klawdek** [comment]
Same thing is happening to me.

**Comment by sainttx** [comment]
Fixed in 07f89c3cfde7ca2e6e83c903660909b1c8f7b03c
